mongodb/doc/MongoRecord.md

```diff
@@ -0,0 +1,25 @@
+# MongoDB Record Dataset
+
+The MongoDB Record Dataset is a write-only
+[dataset](/doc/builtin/datasets/Datasets.md) that writes to a MongoDB
+collection.
+
+Rows are stored in collections to the following format:
+
+```
+{
+    "_id" : <row name>,
+    "<column name>", <column value>,
+    ...
```

Comment:
It is not entirely clear that the ellipsis is replacing other column name, value pairs here.  There is standard to write[ JSON schema](http://json-schema.org/) but it is probably overkilled here.  May be it would be better to simply say that the row _id will become the row name and all the other fields with be columns.